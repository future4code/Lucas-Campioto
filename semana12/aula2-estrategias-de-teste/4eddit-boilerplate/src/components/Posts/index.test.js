import React from 'react'
import {shallow} from 'enzyme'
import renderer from "react-test-renderer";
import {Posts,CardContent} from './index'


describe('test component Posts', () =>{
  

    it('renders correctly', () =>{
        const tree = renderer
            .create(<Posts></Posts>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })

})