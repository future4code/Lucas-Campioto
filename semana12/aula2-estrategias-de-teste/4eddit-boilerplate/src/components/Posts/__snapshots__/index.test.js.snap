// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component Posts renders correctly 1`] = `
<div
  className="sc-fzXfNl AZUci"
>
  <div
    className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <h5
        className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
      />
      <hr />
      <p
        className="MuiTypography-root MuiTypography-body1"
      />
      <hr />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <div
        className="sc-fzXfNJ dICJtc"
      >
        <span
          className="sc-fzXfNL eBJIlm"
        >
          ⬆
        </span>
        <span />
        <span
          className="sc-fzXfNM lfOqWa"
        >
          ⬇
        </span>
      </div>
      <div>
        <span>
          comentários
        </span>
        <span
          className="sc-fzXfNK bQWMbO"
        />
      </div>
    </div>
  </div>
</div>
`;
